ontology/owl/fdri-metadata.ttl

```diff
@@ -328,56 +417,64 @@ above the local reference ellipsoid).""" ;
                        owl:unionOf ( :EnvironmentalMonitoringFacility
                                      :EnvironmentalMonitoringNetwork
                                    )
-                     ] .
+                     ] ;
+          rdfs:label "Relates an Environmental Monitoring Programme to the Environmental Monitoring Facility or Environmental Monitoring Network used by the programme in the delivery of its outputs."@en ,
+                     "utilises"@en .
 
 
 ###  http://fdri.ceh.ac.uk/vocab/metadata/valueStatistic
 :valueStatistic rdf:type owl:ObjectProperty ;
-                rdfs:range :ValueStatistic .
+                rdfs:range :ValueStatistic ;
+                rdfs:comment "Relates an aggregated data resource to the type of aggregation applied."@en ;
+                rdfs:label "value statistic"@en .
 
 
 ###  http://fdri.ceh.ac.uk/vocab/metadata/variable
 :variable rdf:type owl:ObjectProperty ;
-          rdfs:range :Variable .
+          rdfs:range :Variable ;
+          rdfs:comment "Relates a Measure to the Variable that the is measured."@en ;
```

Comment:
s/that the is/that is/

trivial I know, just showing I at least skimmed the details :)

Reply:
Fixed!